packages/integrations/package.json

```diff
@@ -16,13 +16,17 @@
   "module": "esm/index.js",
   "types": "dist/index.d.ts",
   "dependencies": {
-    "@sentry/types": "5.22.3",
-    "@sentry/utils": "5.22.3",
+    "@sentry/angular": "^5.22.2",
+    "@sentry/types": "5.22.2",
+    "@sentry/utils": "5.22.2",
+    "aws-sdk": "^2.741.0",
+    "fs": "0.0.1-security",
```

Comment:
And `fs`

Reply:
made this change in the new pull request.